cluster/addons/metrics-server/metrics-server-deployment.yaml

```diff
@@ -54,17 +54,29 @@ spec:
         command:
```

Comment:
are we sure that this version of the image has the file capabilities on the binary?

Reply:
@dgrisonnet or someone from the metrics servers project can answer this?

Reply:
yes it does, we are running `setcap cap_net_bind_service=+ep metrics-server` on the binary when building the image.

Reply:
the only difference between the job that fails with the binding problem and the job that works, is that the failing one uses dockershim and the good one CRI-> containerd

Reply:
so... does that mean that capabilities are broken in dockershim?

Reply:
ah, ok, my bad, I thought that the deprecation prevaled and that is why I was trying to workaround it ... sorry